pkgs/development/python-modules/flatbuffers/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, buildPythonPackage
+, flatbuffers
+}:
+
+let
+  flatbuf = flatbuffers;
+in buildPythonPackage rec {
+  pname = "flatbuffers";
+  version = flatbuf.version;
+
+  src = flatbuf.src;
```

Comment:
personal preference
```suggestion
buildPythonPackage rec {
  pname = "flatbuffers";
  inherit (flatbuffers) version src;

```

Reply:
Should be resolved in the latest commit.